General documentation improvements (#354)
airbytehq/airbyte: docs/contributing/contributing-to-airbyte/README.md

# Contributing to Airbyte

> We hope you are enjoying Airbyte and we look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we are eager to get  our community input.

Reviewer: ```suggestion
We hope you are enjoying Airbyte and we look forward to hearing your feedback. Whether you want to raise a bug report, a feature/integration request or just request a correction to something, we are eager to get input from our community.
```

## **Requesting new Features**

> You are welcome to create new issues on this project to request new features.

Reviewer: "You are welcome ..." makes it sound non-conclusive. How about: 

```suggestion
To request new features, please create an issue on this project.
```
